### PR TITLE
refactor: ignore resource conponment useless api

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
keys, exec, log command are not supported in cli now 

**Solution:**
Add ignore apis to cli

### CODE REVIEW

**Comment by gitlawr** [comment]
https://github.com/seal-io/walrus/issues/2042

**Review by alexcodelf — APPROVED** [review]
LGTM

File: pkg/apis/runtime/scheme_route_extension.go [hunk 1]
```diff
@@ -37,7 +37,9 @@ var (
 		"/projects/:project/environments/:environment/resources/:resource/revisions/:resourcerevision/terraform-states",
 		"/projects/:project/environments/:environment/resources/_/graph",
 		"/projects/:project/environments/:environment/resources/:resource/graph",
-		"/projects/:project/environments/:environment/resources/:resourcecomponent/keys",
+		"/projects/:project/environments/:environment/resources/:resource/components/:resourcecomponent/keys",
+		"/projects/:project/environments/:environment/resources/:resource/components/:resourcecomponent/log",
+		"/projects/:project/environments/:environment/resources/:resource/components/:resourcecomponent/exec: ",
```
Comment (gitlawr) [inline pkg/apis/runtime/scheme_route_extension.go:42]: What's the purpose of the colon after exec?

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**